Project/Engine/src/sysFrame/System/Json/Json.cpp

```
@@ -29,56 +31,79 @@ void Json::LoadJson(const std::string& _path) {
 
-    Group group;
-	group[_group] = object;
-
-    data.push_back(group);
+Json::Group Json::GetValueArray(const std::string& _path) {
+    auto data = datas_.find(_path);
+    assert(data != datas_.end());
+    return data->second;
 }
 
 Json::Value Json::GetValue(const std::string& _path, const std::string& _group, const std::string& _key) const {
```

Cppcheck (2.10-2)

[style] The function 'GetValue' is never used.

(unusedFunction)